### ISSUE TITLE
Fasttext word vectors: Why English word vectors yield better results?

### ISSUE THREAD

**Issue by payman-kh** [facebookresearch/fastText#1037]
I have been working on a text classification task with texts in English, German and French. I trained the same neural network for each language using the pre-trained word vectors provided by Fasttext (so English, German and French word-vectors were used).
But the accuracy of the English model (for English texts) is always higher by at least 10 percent than the German and French models. 
How can this be explained? is it because the pre-trained word-vectors for English are somehow "better" or more "accurate" compared to word vectors in other languages? Or is it not necessarily related to fasttext models?

**Comment by payman-kh** [comment]
.
